cmd/kops/integration_test.go

```diff
@@ -80,6 +80,11 @@ func TestComplex(t *testing.T) {
 	runTestAWS(t, "complex.example.com", "complex", "legacy-v1alpha2", false, 1, true, nil)
 }
 
+// TestCrossZone tests that the the cross zone setting on the API ELB is set properly
+func TestCrossZone(t *testing.T) {
+	runTestAWS(t, "crosszone.example.com", "api_elb_cross_zone", "v1alpha2", false, 1, true, nil)
```

Comment:
The function signature for `runTestAWS` recently changed in the master branch. You'll want to rebase your branch and add the new boolean argument. Alternatively you can follow justinsb's advice and add the setting to the existing `complex` integration test.